install.sh

```diff
@@ -709,9 +714,25 @@ install_buildah(){
   log::info "Installing Buildah and configuring user namespaces..."
 
   case "$distro" in
-    *Debian*|*Ubuntu*)
+    *Debian*)
       install_package "buildah uidmap"
       ;;
+    *Ubuntu*)
+      VERSION_ID=$(lsb_release -r | cut -f2)
+      if [[ "$VERSION_ID" == "20.04" || "$VERSION_ID" == "18.04" ]]; then
+      install_package "wget ca-certificates gnupg2"
+      echo "deb http://download.opensuse.org/repositories/devel:/kubic:/libcontainers:/stable/xUbuntu_${VERSION_ID}/ /" | run_as_root "tee /etc/apt/sources.list.d/devel-kubic-libcontainers-stable.list"
+      curl -Ls https://download.opensuse.org/repositories/devel:kubic:libcontainers:stable/xUbuntu_$VERSION_ID/Release.key | run_as_root "apt-key add -"
+      run_as_root "apt-get update"
+      install_package "libfuse3-dev fuse-overlayfs"
+      fi
+      install_package "buildah uidmap"
+
+      # workaround for https://github.com/containers/podman/issues/11745
+      if [[ "$VERSION_ID" == "20.04" || "$VERSION_ID" == "18.04" ]]; then
```

Comment:
Even in the issue you mentioned it's not about Ubuntu, but Arch. Looks like it's a more widespread problem. Would be better to do this:
1. Check if podman/buildah/whatever are recent enough to support the `machine` instruction.
1. If not, then check for `machine` and remove it if needed.

P.S.: do we really need this? Looks like this is just warnings, so it shouldn't break anything. Because changing `/usr/share/containers/containers.conf` is not gonna be permanent, and after updating podman the issue will probably return. There are workarounds like make the file read-only or something, so that it cannot be changed/replaced, but there might be problems during updates then.